src/peft/mapping.py

```diff
@@ -181,6 +182,21 @@ def get_peft_model(
     new_name = model.__dict__.get("name_or_path", None)
     peft_config.base_model_name_or_path = new_name
 
+    # Especially in notebook environments there could be a case that a user
```

Comment:
We have 120 char limits in the project, could you please configure your formatter accordingly?